src/main/java/org/isf/medicalinventory/gui/InventoryWardBrowser.java

```diff
@@ -157,7 +217,7 @@ private JPanel getPanelHeader() {
             gbc_stateLabel.insets = new Insets(0, 0, 0, 5);
             gbc_stateLabel.gridx = 4;
             gbc_stateLabel.gridy = 0;
-            panelHeader.add(getStateLabel(), gbc_stateLabel);
+            panelHeader.add(getStatusLabel(), gbc_stateLabel);
```

Comment:
if you are going to change `State` to `Status` then do it everywhere:  `gbc_stateLabel` ==> `gbc_statusLabel`